### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by drjsanger**
@keiranmraine Could I get an initial critique please?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,25 +20,30 @@ Contents:
...
 There are pre-built images containing this codebase on quay.io.
 
+* [cgpPindel][cgpPindel-git]: Contained within this repository - contains the cgpPindel package
 * [dockstore-cgpwxs][ds-cgpwxs-git]: Contains tools specific to WXS analysis.
 * [dockstore-cgpwgs][ds-cgpwgs-git]: Contains additional tools for WGS analysis.
 
 These were primarily designed for use with dockstore.org but can be used as normal containers.
 
-The docker images are know to work correctly after import into a singularity image.
+The docker images know to work correctly after import into a singularity image.
```
Comment (keiranmraine) [inline README.md]: I think thats the wrong edit.  Pretty sure it should have read:

_The docker images **are known** to work correctly after import into a singularity image._

Comment (drjsanger) [reply README.md]: Fixed.

**Comment by keiranmraine** [comment]
Hi @drjsanger 

The Dockerfile should be sending copying the current workspace into the build and not pulling the pindel code from github as part of `opt-build.sh`.

It's as simple as replacing line [24](https://github.com/cancerit/cgpPindel/pull/69/files#diff-3254677a7917c6c01f55212f86c57fbfR24) with:

```
COPY . .
```

before executing the build script and updating it to use the local code instead of pulling it.

**Comment by keiranmraine** [comment]
@drjsanger I've applied the remaining changes to this branch can you give it a ready through?  Approved I'll set up quay.io to build when tagged (and on demand)

**Pull request merged**